main.go

```diff
@@ -447,6 +454,18 @@ func main() {
 		log.Fatal("error: missing eth-rpc parameter")
 	}
 
+	if len(vestingContractAddress) <= 0 {
+		log.Fatal("error: missing vesting-contract-address parameter")
+	}
+
+	if len(deploymentHeightStr) <= 0 {
+		log.Fatal("error: missing deployment-height parameter")
+	}
+
+	if len(deploymentHeightStr) <= 0 {
+		log.Fatal("error: missing deployment-height parameter")
+	}
```

Comment:
Why check it twice?

Reply:
Removed entirely as there are now defaults